gnpy/core/service_sheet.py

```diff
@@ -105,7 +111,9 @@ def __init__(self,Request,eqpt_filename):
             # print(msg)
 
         self.loose = 'loose'
+        print(Request.is_loose)
```

Comment:
What are the ranges of values? 

```
Request.is_loose in {'loose', 'no'} # what other possibilities?
```

Reply:
In fact I had in mind **'loose' and 'strict',** but I think I mixed up a bit with the excel format. right now the excel sheet should have a yes or no in 'routing: is loose ?' column, but the code uses loose and strict. Moreover, somebody asked me for a stricter constraint, which would be 'use exactly the succession of node' as a path, so there might be an 'exact' term in a near future if this proves to be useful.